solr/solrj/src/java/org/apache/solr/common/util/IOUtils.java

```diff
@@ -31,7 +32,7 @@ public static void closeQuietly(Closeable closeable) {
         closeable.close();
       }
     } catch (Exception e) {
-      LOG.error("Error while closing", e);
+      SolrException.log(LOG, "Error while closing "+closeable.getClass());
```

Comment:
I had to modify the logging here in order to have `ignoreException` working in the tests.